spirv_reflect.c

```diff
@@ -2448,6 +2477,18 @@ static SpvReflectResult ParseDescriptorBlockVariable(
             }
           }
           if (!found_recursion) {
+            uint32_t struct_id = FindType(p_module, p_member_type->id)
+                                     ->struct_type_description->id;
+            p_parser
```

Comment:
@chaoticbob I'm going to just make a PR to run clang-format with more than 80 lines as this starts to just become unnecessarily hard to read